### DISCUSSION TITLE
[Projects Beta] Convert to issue list of projects should remember recent selections

### DISCUSSION THREAD

**Discussion by MarkBennett** [community/community#10132] [Projects and Issues]
I have a lot of repositories (approaching 200), and when converting an item to an issue it always shows me the same list of repositories to attach it to. These don't seem to have any relation to the repositories I've attached issues to in the past, other repositories in use on the board, or repositories with recent activity which means I have to search for my repository *a lot*.

So that I don't need to search for repositories when converting an item to an issue, the repository picker should factor in personalized recommendations based on recently used repos, repos in use on the board, and github activity elsewhere.

**Comment by sammarks** [comment]
Agreed; for the record, it appears this worked in the past but was broken with a recent change? Not sure what the logic was around it, but it would always display only repositories that had issues inside the current view it seemed.

Now, it just shows my account repositories, in no obvious order.

**Comment by DennisSmolek** [comment]
My problem is that I've added the project to a single repo, it shows on that repo's project page, but it doesn't show up in the list of repos..

Now, I did everything kinda quick so maybe it's a time thing..

 or maybe I did something wrong related to public/private.
But if that's the case there should be some sort of notice

**Reply by DennisSmolek** [reply]
Figured this out.

When forking an existing repo the "issues" feature is disabled by default (was for this repo anyway)

Because of this projects correctly doesn't list the repo in the list..

### Suggestion:
1. If the Project is on a repo with issues disabled, send a notice/warning/etc 

2. After the default (correct) list of projects, list other projects on the user. Can grey them out, strikethrough, etc. at least tell me you see the project and it's not some weird error. Even better would be a hint like "Issues not enabled" "Visibility Missmatch" "permissions error" etc. I'm sure theres a filter/processor that adds these flags in the search function.
Similar to Mac for "Open With" or similar context menus.
